template/node/Cargo.toml

```diff
@@ -89,6 +89,7 @@ default = ["with-rocksdb-weights"]
 with-rocksdb-weights = ["frontier-template-runtime/with-rocksdb-weights"]
 with-paritydb-weights = ["frontier-template-runtime/with-paritydb-weights"]
 rpc-binary-search-estimate = ["fc-rpc/rpc-binary-search-estimate"]
+txpool = []
```

Comment:
you also need to add feature into `fp-rpc`, `fc-rpc` and `fc-rpc-core` .

Reply:
done.